Automattic/wp-calypso
# Jetpack Licensing: Add Total Cost to Issue License Page

#### Proposed Changes

This PR adds the total cost to `/partner-portal/issue-license`.

<img width="1069" alt="image" src="https://user-images.githubusercontent.com/42627630/202331766-c18b75ca-6d47-478d-b1c0-19c8114682f1.png">

<img width="388" alt="image" src="https://user-images.githubusercontent.com/42627630/202830799-efa8dd84-0e55-43f9-a47c-a44ab25f957e.png">


#### Testing Instructions



1. Visit [/partner-portal/issue-license](http://jetpack.cloud.localhost:3001/partner-portal/issue-license). 
2. Click on various licenses. Verify that the total cost displayed is correct according to what is selected.
3. Test that the UI looks good in both desktop and mobile views.

#### Pre-merge Checklist





Related to 1203126240279377-as-1203126240279434
